upup/pkg/fi/cloud.go

```diff
@@ -16,7 +16,11 @@ limitations under the License.
 
```

Comment:
I think there are a lot of edge cases, so it would be good to add tests, yes!
- What if there is no ip range available for new subnets in the vpc?
- What if the leftover ip range in the vpc is so small that it's not possible to create 3 subnets in it?
- What if the 3 subnets that can be created are so small that it's not worth creating? (Ex: /32 range...)
- What if the existing subnets leave some free Ip ranges only in between them...)
- Etc